### ISSUE TITLE
Integrate the bot and web app more

### ISSUE THREAD

**Issue by deadbender** [penny-university/penny_university#372]
# Concepts

* /penny help redirects to a help page showing all possible commands
* /penny profile provides a link to the user's profile
* /penny chats provides a link to pennyu.org/chats

Button linking to each new URL

**Comment by nchouard** [comment]
For `/penny help`, I would just return the list of commands in Slack, which then could lead them to use the others to get into the web app. It would be a strange UX to be taken out of slack for that when I'm wanting to find out what the slack app itself can do.

**Comment by JnBrymn** [comment]
I agree w/ @nchouard regarding `/penny help`. The functionality of the others I think is great. I've actually needed it quite a bit. But I would recommend consolidating. Rather than `/penny chats` and `/penny profile` what if we just had `/penny chats` which could optionally be followed by a user name.

* `/penny chats` - goes to the all chats page
* `/penny chats @nchouard` - goes to Nick's chats
* `/penny chats @me` (I just checked... this works in slack) - goes to my chats

I like consolidating in that it's one less command. However it does inconsistently go to the profile page some times and to the chats page some times. That's a little annoying.

**Comment by mrshwah** [comment]
I like that idea @JnBrymn.

**Comment by JnBrymn** [comment]
# getting started
* [Set up your dev environment.](https://github.com/penny-university/penny_university/blob/master/docs/development_environment.md)
* [When you type /penny chat in slack, slack sends the information here.](https://github.com/penny-university/penny_university/blob/dcd873d7123940eabff8cbd6f5d110763ec2d9fe/bot/views.py#L63)
  * You'll probably want to create a `chats` elif clause and a new `BotModule` kinda like `PennyChatBotModule` ([which is defined here](https://github.com/penny-university/penny_university/blob/87a297e008d74209c07e998938959da21044d229/bot/processors/pennychat.py#L307))
